charts/consul/templates/crd-servicedefaults.yaml

```diff
@@ -8,8 +7,8 @@ metadata:
   creationTimestamp: null
   name: servicedefaults.consul.hashicorp.com
   labels:
-    app: {{ template "consul.name" . }}
-    chart: {{ template "consul.chart" . }}
+    app: consul
+    chart: consul-helm
```

Comment:
The changes to this file should be autogenerated by running `make ctrl-generate ctrl-manifests`

Reply:
@thisisnotashwin Thanks for the comments!! Auto-gen code is cool
After I run `make ctrl-generate ctrl-manifests`,  there are few other files got changed as well
```
	modified:   charts/consul/templates/crd-peeringacceptors.yaml
	modified:   charts/consul/templates/crd-peeringdialers.yaml
	modified:   charts/consul/templates/crd-servicedefaults.yaml
	modified:   control-plane/api/v1alpha1/zz_generated.deepcopy.go
	modified:   control-plane/config/crd/bases/consul.hashicorp.com_servicedefaults.yaml
```
Could you help double check if the changes are intended?